test/security_tls_cipherlists.bats

```diff
@@ -19,8 +19,8 @@ function setup_file() {
     # Shared config for TLS testing (read-only)
     export TLS_CONFIG_VOLUME
     TLS_CONFIG_VOLUME="$(pwd)/test/test-files/ssl/${DOMAIN}/:/config/ssl/:ro"
-    # `${BATS_TMPDIR}` maps to `/tmp`
-    export TLS_RESULTS_DIR="${BATS_TMPDIR}/results"
+    # `${TMPDIR}` maps to `/tmp`
+    export TLS_RESULTS_DIR="/tmp/results"
```

Comment:
```suggestion
    export TLS_RESULTS_DIR="${RUNNER_TEMP}/results"
```

Reply:
one issue i could find is that this directory (`TLS_RESULTS_DIR` no matter where it's located on the system) is not cleaned with the other test data in `make clean` (not sure if that's the reason)